src/Field.js

```diff
@@ -54,8 +54,10 @@ const createField = ({ deepEqual, getIn }) => {
     }
 
     render() {
-      const { ConnectedField } = this
-      return <ConnectedField {...this.props} ref="connected"/>
+      return createElement(this.ConnectedField, {
+        ...this.props,
+        ref: 'connected'
+      })
```

Comment:
I replaced JSX in favor of `createElement` to take control over what transpilation is doing. The end result is cleaner in my opinion because by reading the code we immediately understand what react is doing. I've done these changes in a separate commit so it could be easily reverted.

Note: It is also how `react-redux` instantiate React elements.